module/VuFind/src/VuFind/Session/Memcache.php

```diff
@@ -81,10 +81,10 @@ protected function connect(?Config $config, ?object $client): void
         $host = $config->memcache_host ?? 'localhost';
         $port = $config->memcache_port ?? 11211;
         $timeout = $config->memcache_connection_timeout ?? 1;
-        $clientClass = $config->memcache_client ?? 'Memcache';
+        $clientClass = $config->memcache_client ?? \Memcache::class;
```

Comment:
We can't change the default here or it will break things. We can simply phase out the memcache_client setting entirely in the next major version.